test/e2e/gc_e2e_test.go

```diff
@@ -284,4 +284,101 @@ var _ = Describe("Garbage collection for dependent resources", func() {
 
 	})
 
+	Context("When installing a bundle with configmap and secret objects these objects should be deleted when the CSV is deleted", func() {
+		const (
+			sourceName    = "test-catalog"
+			imageName     = "quay.io/olmtest/single-bundle-index:objects"
+			packageName   = "busybox"
+			channelName   = "alpha"
+			subName       = "test-subscription"
+			csvName       = "busybox.v2.0.0"
+			secretName    = "mysecret"
+			configmapName = "special-config"
+		)
+
+		var (
+			source *v1alpha1.CatalogSource
+			err    error
```

Comment:
We only want to declare variables at Container level only if we want to share them between multiple `It` specs. err isn't something we would want to share across multiple specs. So it is advisable to declare it right before it is used. [See here](https://github.com/operator-framework/operator-lifecycle-manager/blob/db290a7d8503dcfa8f49cb976bcc3da1b054392b/test/e2e/metrics_e2e_test.go#L60)